common/log/v2ray/logger.go

```diff
@@ -0,0 +1,67 @@
+package v2ray
+import (
+	"fmt"
+	"os"
+	vlog "v2ray.com/core/common/log"
+
+	"github.com/eycorsican/go-tun2socks/common/log"
+)
+
+type v2rayLogger struct {
+	handler vlog.Handler
+	level   vlog.Severity
+}
+
+func NewV2RayLogger() log.Logger {
+	handler := vlog.NewLogger(vlog.CreateStdoutLogWriter())
+	return &v2rayLogger{handler: handler, level: vlog.Severity_Info}
+}
+
+func (l *v2rayLogger) SetLevel(level log.LogLevel) {
+	switch level {
+	case log.DEBUG:
+		l.level = vlog.Severity_Debug
+	case log.INFO:
+		l.level = vlog.Severity_Info
+	case log.WARN:
+		l.level = vlog.Severity_Warning
+	case log.ERROR:
+		l.level = vlog.Severity_Error
+	case log.NONE:
+		l.level = vlog.Severity_Unknown
+	}
+}
+
+func (l *v2rayLogger) Debugf(msg string, args ...interface{}) {
+	l.output(vlog.Severity_Debug, msg, args)
```

Comment:
`args` should be unpacked, i.e. `args...`

Reply:
Fixed in new commit